### PR TITLE
ISPN-6412 Make RESTReplicationIT stable

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/ISPN-6412

As @danberindei suggested [here](https://issues.jboss.org/browse/ISPN-6412?focusedCommentId=13188829&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13188829) I added clean up code to all places where `RESTHelper` had been used.

### CODE REVIEW

**Comment by galderz** [comment]
Needs rebasing

**Comment by galderz** [comment]
Otherwise the PR looks good

**Comment by danberindei** [comment]
ExpirationIT and ExampleConfigsIT fail with "java.lang.Exception: Method clearServers() should be static"

File: server/integration/testsuite/src/test/java/org/infinispan/server/test/rollingupgrades/RestRollingUpgradesDistIT.java [hunk 1]
```diff
@@ -42,6 +43,11 @@
     @ArquillianResource
     ContainerController controller;
 
+    @AfterClass
+    public void clearServers() {
```
Comment (danberindei) [inline server/integration/testsuite/src/test/java/org/infinispan/server/test/rollingupgrades/RestRollingUpgradesDistIT.java]: Should be static, same as the other `@AfterClass` methods

**Comment by slaskawi** [comment]
Rebased and comments addressed

**Comment by slaskawi** [comment]
Rebased

**Comment by pruivo** [comment]
looks good to me. @galderz @danberindei any other comments?

**Comment by danberindei** [comment]
No more comments @pruivo

**Comment by pruivo** [comment]
this was already integrated. closing it. thanks @slaskawi !

**Pull request closed without merging**